Specs/Scene/BillboardCollectionSpec.js

```diff
@@ -785,6 +787,25 @@ defineSuite([
         expect(scene.renderForSpecs()).not.toEqual([0, 0, 0, 255]);
     });
 
+    it('renders billboards when instancing is disabled', function() {
```

Comment:
Check code coverage (see [here](https://github.com/AnalyticalGraphicsInc/cesium/wiki/Contributor%27s-Guide#buildtargets).  I think we will need a few more tests to coverage all the billboard paths.